README.md

```diff
@@ -270,77 +270,29 @@ For testing the API keys, please refer to this [documentation](./docs/api_key.md
 
 ### Run
 
-There are multiple ways in which we can now use the neuro-san server with a client:
+Neuro SAN Studio provides a user-friendly environment to interact with agent networks.
 
-<!-- pyml disable-next-line line-length -->
-#### Option 1: Using [`nsflow`](https://github.com/cognizant-ai-lab/nsflow) as a developer-oriented web client
+1. Start the server and client with a single command, from the project root directory:
 
-If you want to use neuro-san with a FastAPI-based developer-oriented client, follow these steps:
-
-* Start the server and client with a single command, from project root:
-
-  ```bash
-  python -m run
-  ```
-
-* As a default
-    * Frontend will be available at: `http://127.0.0.1:4173`
-    * The client and server logs will be saved to `logs/nsflow.log` and `logs/server.log` respectively.
-
-* To see the various config options for this app, on terminal
-
-  ```bash
-  python -m run --help
-  ```
-
-Screenshot:
-
-![NSFlow UI Snapshot](https://raw.githubusercontent.com/cognizant-ai-lab/nsflow/main/docs/snapshot01.png)
-
----
+    ```bash
+    python -m run
+    ```
 
-#### Option 2: Using a basic web client interface
+2. Navigate to [http://localhost:4173/](http://localhost:4173/) to access the UI.
+3. (Optional) Check the logs:
+   * For the server logs: `logs/server.log`
+   * For the client logs: `logs/nsflow.log`
+   * For the agents logs: `logs/thinking_dir/*`
 
-A [basic web client interface](https://github.com/cognizant-ai-lab/neuro-san-web-client) is installed by default.
-It's a great, simple example of how to connect to a neuro-san server and interact with it.
-Start the server and the client in one single command:
+Use the `--help` option to see the various config options for the `run` command:
 
 ```bash
-python -m run --use-flask-web-client
+python -m run --help
 ```
 
-The client and server logs will show on the screen,
```

Comment:
Moved that to the README of the [web client repo](https://github.com/cognizant-ai-lab/neuro-san-web-client) (PR not ready yet)